.gitignore

```
@@ -0,0 +1,4 @@
+# Ignore all compiled Python files in the click and pyotp directories
+click/__pycache__/*
+pyotp/__pycache__/*
+pyotp/contrib/__pycache__/*
```

```suggestion
__pycache__/
```

Please ignore globably instead.